src/App.js

```diff
@@ -1,12 +1,22 @@
 import React from 'react'
+import {Route, Routes} from "react-router-dom";
 import './App.css';
 import Calculator from './Components/Calculator';
+import Header from './Components/Header'
+import Home from './Components/Home'
+import Quote from './Components/Quote.js';
+
 
 class App extends React.Component {
   render() {
     return (
       <div className="App">
-        <Calculator />
+        <Header />
+        <Routes>
+          <Route path='/Home' element={<Home />}></Route>
```

Comment:
- [ ] Your **Home** component should be on the default route means instead of 
**<Route path='/Home' element={<Home />}></Route>** 
Put this :arrow_down:
**<Route path='/' element={<Home />}></Route>** 
at the path in line 16 :blush: